base/commandLine.cpp

```diff
@@ -1039,8 +1175,15 @@ bool processSettings(Common::String &command, Common::StringMap &settings, Commo
 		return true;
 	} else if (command == "auto-detect") {
 		// If auto-detects succeed, we want to return false so that the game is started
-		return !autoDetect(settings["path"]);
+		return !detectGames(settings["path"], true);
 		//return true;
+	} else if (command == "detect") {
+		detectGames(settings["path"], false);
+		return true;
+	} else if (command == "add") {
+		return !addGame(settings["path"]);
```

Comment:
Do we actually want to start ScummVM after adding a game with the command line?
The same question also applies to mass add below.

I don't have a definite answer to this question. But I think I would not expect ScummVM to start after adding the game(s).